internal/kafka/command_broker_describe.go

```diff
@@ -45,15 +30,15 @@ func (c *brokerCommand) newDescribeCommand() *cobra.Command {
 	}
 
 	cmd.Flags().Bool("all", false, "Get cluster-wide broker configurations (non-default values only).")
-	cmd.Flags().String("config-name", "", "Get a specific configuration value (pair with --all to see a cluster-wide configuration.")
+	cmd.Flags().String("config-name", "", "Get a specific configuration value (pair with --all to see a cluster-wide configuration).")
```

Comment:
Good catch here. We can't use backticks here due to a limitation in Cobra, but regular quotes are better than nothing.
```suggestion
	cmd.Flags().String("config-name", "", `Get a specific configuration value (pair with "--all" to see a cluster-wide configuration).`)
```